// [START declaration]
syntax = "proto3";
package aiconnector;
// [END declaration]

// [START messages]
message MoveMessage {

    enum RequestType {
        GET_MOVE = 0;
        SET_MOVE = 1;
    }

    enum ResponseType {
        MOVE  = 0;
        OK    = 1;
        ERROR = 2;
    }

    message Move {
        string name = 1;
    }

    RequestType  requesttype = 1;
    ResponseType responsetype = 2;
    Move         move = 3;
}
// [END messages]
